Assets/Scripts/Utils/Enums.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Utils
+{
+    public class Enums : MonoBehaviour
```

Comment:
We should have different classes for each type of enum. Probably better to make it directly be enum StatType

Reply:
I don't get it

Reply:
Don't use an Enum-Class, make an enum directly:
`public enum StatTypes`
And I would probably make this an inner enum as well. It is only related to the stats and will never be used in other context.

Reply:
okay I put it in SanityController